ui/dialog.js

```diff
@@ -790,6 +786,7 @@ return $.widget( "ui.dialog", {
 				.appendTo( iframe.parent() )
 				.offset( iframe.offset() )[0];
 		});
+		return this;
```

Comment:
Do we really want to support chaining instance methods in more places? Most `$.Widget` methods currently don't support chaining, most dialog methods don't support it either. Making a select few methods chainable just because its convenient in a few places seems questionable.